client/components/jetpack/daily-backup-status/status-card/backup-tips.tsx

```diff
@@ -0,0 +1,59 @@
+import { useTranslate } from 'i18n-calypso';
+import DotPager from 'calypso/components/dot-pager';
+
+import './style.scss';
+
+type BackupTipsProps = {
+	location: 'IN_PROGRESS' | 'SCHEDULED' | 'COMPLETED' | 'NO_BACKUPS';
+};
+
+// locations where the message about the initial backup should be shown
+const LOCATIONS_4_INITIAL_BACKUP_TIP: Array< BackupTipsProps[ 'location' ] > = [
```

Comment:
Very very minor but other constants in the codebase typically uses the fully spelled out "\_FOR\_", i.e. `READY_FOR_UPLOAD` and `REASONS_FOR_MANUAL_REVIEW`.

Reply:
Oh, thank you for pointing that out. I just wanted to make the name a bit shorter 😄 

Will update.